Libplanet/Store/Trie/MerkleTrie.cs

```diff
@@ -91,6 +91,51 @@ public ITrie Commit()
             return new MerkleTrie(KeyValueStore, newRoot);
         }
 
+        internal IEnumerable<HashDigest<SHA256>> IterateHashNodes()
+        {
+            if (Root is null)
+            {
+                yield break;
+            }
+
+            var queue = new Queue<HashNode>();
+            if (!(Root is HashNode))
+            {
+                yield return Root.Hash();
+            }
+            else
+            {
+                queue.Enqueue((Root as HashNode)!);
+            }
+
+            while (queue.Count > 0)
+            {
+                var hashNode = queue.Dequeue();
+                yield return hashNode.HashDigest;
+                var node = GetNode(hashNode.HashDigest);
```

Comment:
```suggestion
                HashNode node = GetNode(hashNode.HashDigest);
```

Reply:
Actually, its type is `INode?`, not `HashNode`. Your intention seems that let's use without using `var` keyword in the section, so I will apply it with `INode?` type.

Reply:
🆗